packages/graphql_flutter/example/graphql_chat/lib/view/home/home_body.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:graphql_chat/api/query.dart';
+import 'package:graphql_chat/model/chat.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
+import 'package:logger/logger.dart';
+
+class HomeViewBody extends StatelessWidget {
+  final Logger _logger = Logger();
+  List<Chat> _chats = [];
+
+  HomeViewBody({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const SizedBox(width: 20, height: 90),
+        Expanded(child: _buildBodyView(context: context))
+      ],
+    );
+  }
+
+  Widget _buildScrollView({required BuildContext context}) {
+    return Query(
```

Comment:
IMO, I think we should advocate for hooks for readability

Reply:
Yes, my idea was making multiple example of the same app and see how it is easy use the hooks, and also this is useful at migration time, what do you think?